backend/internal/handlers/classrooms/assignments/works/works.go

```diff
@@ -64,12 +75,71 @@ func (s *WorkService) getWorksInAssignment() fiber.Handler {
 		if err != nil {
 			return err
 		}
+
+		// get list of students in class to get which students havent accepted the assignment
+		students, err := s.store.GetUsersInClassroom(c.Context(), int64(classroomID))
```

Comment:
Isn't this a list of everyone in the class, not just students

Reply:
I will fix this, but this will also make manual testing slightly more difficult